MemberExpectations.md

```diff
@@ -1,7 +1,7 @@
-It is important that members of our leadership groups (TSC, CommComm) act
-in a way that not only complies with the
-[Code of Conduct](CODE_OF_CONDUCT.md)
-but that supports the vision, mission and operating principles of the Node.js project.
```

Comment:
Are there any documents we can link to for "vision, mission, and principles"?

Reply:
Probably not. Even if there are docs, it feels like it's just cover for what's to follow. Might as well just get to what's to follow.